ubtue_Brill.js

```diff
@@ -76,8 +76,10 @@ function postProcess(doc, item) {
 		 item.abstractNote = '';
 	}
 	item.tags = ZU.xpath(doc, '//dd[contains(@class, "keywords")]//a');
-	if (item.tags)
-		item.tags = item.tags.map(i => i.textContent.trim());
+	if (item.tags) {
+		let deduplicateTags = item.tags.map(i => i.textContent.trim());
```

Comment:
Maybe rather `allTags` instead of `deduplicateTags` and short comment `//deduplicate` in the next line?